automation/Zhp.Office.AccountManagement/Zhp.Office.AccountManagement.Tests/Domain/Model/ActivationRequestTests.cs

```diff
@@ -0,0 +1,61 @@
+using FluentAssertions;
+using Xunit;
+using Zhp.Office.AccountManagement.Model;
+
+namespace Zhp.Office.AccountManagement.Tests.Domain.Model;
+
+public class ActivationRequestTests
+{
+    [Theory]
+    [InlineData("Jan", "Jan")]
+    [InlineData("ANNA", "Anna")]
+    [InlineData("renée", "Renée")]
+    [InlineData("ToMaSz", "Tomasz")]
+    [InlineData("bARTŁOMIEJ", "Bartłomiej")]
+    [InlineData("JAN KANTY", "Jan Kanty")]
+    [InlineData("øyvind", "Øyvind")]
+    [InlineData("JEAN-BAPTISTE", "Jean-Baptiste")]
+    [InlineData("von vordon", "Von Vordon")] // acceptable, but wrong
```

Comment:
tu jest taka trudność, że pewne części nazwisk jak _von_ czy _de_ powinny być z małych liter, ale to jest taki edge case, że bym olał